client/app/bundles/course/level/pages/LevelsIndex/LevelsManager.tsx

```diff
@@ -0,0 +1,150 @@
+import { useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { Button, Slide, Typography } from '@mui/material';
+
+import { defaultComponentTitles } from 'course/translations.intl';
+import Page from 'lib/components/core/layouts/Page';
+import { useAppDispatch, useAppSelector } from 'lib/hooks/store';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import LevelsTable from '../../components/LevelsTable';
+import { saveLevels } from '../../operations';
+import { getLocalState } from '../../selector';
+
+const translations = defineMessages({
+  unsavedChanges: {
+    id: 'course.level.Level.unsavedChanges',
+    defaultMessage: 'You have unsaved changes',
+  },
+  saveChanges: {
+    id: 'course.level.Level.saveChanges',
+    defaultMessage: 'Save',
+  },
+  reset: {
+    id: 'course.level.Level.reset',
+    defaultMessage: 'Reset',
+  },
+  saveSuccess: {
+    id: 'course.level.Level.saveSuccess',
+    defaultMessage: 'Levels Saved',
+  },
+  saveFailure: {
+    id: 'course.level.Level.saveFailure',
+    defaultMessage: 'Failed to save levels',
+  },
+});
+
+const LevelsManager = (): JSX.Element => {
+  const { canManage, levels } = useAppSelector(getLocalState);
```

Comment:
Don't use `getLocalState` over here; there are lots of occasions in all the `store.ts` over our codebase that use this variable. Use something like `getLevels` instead

Refer to the other `store.ts` on how to implement them

Reply:
Done